### DISCUSSION TITLE
Delphi DFM plugin

### DISCUSSION THREAD

**Discussion by casetalk** [WinMerge/winmerge#1322] [Q&A]
After having used WinMerge for many years while developing in Delphi myself, it is only by accident I discovered the Delphi(4!) source code for the Delphi DFM file support. I fixed the source code in Delphi 11, and tested in with the latest WinMerge. It works as needed, preserving the original file format when re-packing the changes.

I've cloned the repository and invite you to pick up on the sources / binaries. If I missed anything (being a git noob) let me know.

**Comment by sdottaka** [comment]
Pull requests are welcome
